src/main/java/uk/gov/justice/probation/courtcasematcher/model/offendersearch/MatchType.java

```diff
@@ -24,10 +40,12 @@ public static MatchType of(OffenderSearchMatchType offenderSearchMatchType) {
                 return PARTIAL_NAME;
             case PARTIAL_NAME_DOB_LENIENT:
                 return PARTIAL_NAME_DOB_LENIENT;
+            case EXTERNAL_KEY:
+                return EXTERNAL_KEY;
```

Comment:
I did consider whether to return EXTERNAL_KEY_PNC so that we're future proofed against if we start matching on other identifiers like CRO but decided against as that's not on our radar at all and we could always do a data migration in court-case-service if we did want to add this. I'd be happy either way though.

Reply:
Yes - it's not on the radar so keep it as it is I think.